<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>764</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Inference System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1,1">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>13</number>
      </property>
      <property name="verticalSpacing">
       <number>17</number>
      </property>
      <item row="2" column="2">
       <widget class="QPushButton" name="add_2">
        <property name="text">
         <string>Add Inference</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="calculate">
        <property name="text">
         <string>Execute Command</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="add">
        <property name="text">
         <string>Add Truth</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="diminish">
        <property name="text">
         <string>Diminsh Truth</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="7">
       <widget class="QListView" name="Input_1">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="diminish_2">
        <property name="text">
         <string>Diminish Infer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Current Rule:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="7">
       <widget class="QListView" name="Input_2">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>55</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="Infer">
        <property name="text">
         <string>Infer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>24</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="1" rowspan="2">
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>28</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>InputBox:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="output">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionFrom_DataBase"/>
     <addaction name="actionFrom_File"/>
    </widget>
    <widget class="QMenu" name="menuSave_As">
     <property name="title">
      <string>Save As...</string>
     </property>
     <addaction name="actionAs_File"/>
     <addaction name="actionAs_File_2"/>
    </widget>
    <addaction name="actionLink_MySql"/>
    <addaction name="menuOpen"/>
    <addaction name="menuSave_As"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Inference_System"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <widget class="QMenu" name="menuSelect_Language">
     <property name="title">
      <string>Select Language</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionEnglish_US"/>
     <addaction name="actionChinese"/>
    </widget>
    <addaction name="menuSelect_Language"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFrom_DataBase">
   <property name="text">
    <string>From DataBase</string>
   </property>
  </action>
  <action name="actionFrom_File">
   <property name="text">
    <string>From File</string>
   </property>
  </action>
  <action name="actionAs_File">
   <property name="text">
    <string>As DataBase</string>
   </property>
  </action>
  <action name="actionAs_File_2">
   <property name="text">
    <string>As File</string>
   </property>
  </action>
  <action name="actionAbout_Inference_System">
   <property name="text">
    <string>About Inference System</string>
   </property>
  </action>
  <action name="actionEnglish_US">
   <property name="text">
    <string>English(US)</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>Chinese(CN)</string>
   </property>
  </action>
  <action name="actionLink_MySql">
   <property name="text">
    <string>Link MySql</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
